filter-expSmooth.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __FILTER_EXP_SMOOTH__
+#define __FILTER_EXP_SMOOTH__
+
+int filterExpSmooth()
+{
+  double alp = 0.5;
+  static int prev_filtered_sample = -1;
+  if (prev_filtered_sample < 0)
```

Comment:
this is strange
letus avoid static variables - it is not straightforward enough what is the value of each
this can easily be avoided by passing prev va;ue in argument ar smth like that